src/sentry/snuba/events.py

```diff
@@ -13,222 +13,278 @@ class Columns(Enum):
-    SDK_NAME = Column("events.sdk_name", "sdk_name", "sdk_name", "sdk.name")
-    SDK_VERSION = Column("events.sdk_version", "sdk_version", "sdk_version", "sdk.version")
+    EVENT_ID = Column("events.event_id", "event_id", "event_id", "event_id", "id")
+    GROUP_ID = Column("events.group_id", "group_id", None, "group_id", "issue.id")
+    PROJECT_ID = Column("events.project_id", "project_id", "project_id", "project_id", "project.id")
+    TIMESTAMP = Column("events.timestamp", "timestamp", "finish_ts", "timestamp", "timestamp")
+    TIME = Column("events.time", "time", "bucketed_end", "time", "time")
+    CULPRIT = Column("events.culprit", "culprit", None, "culprit", "culprit")
+    LOCATION = Column("events.location", "location", None, "location", "location")
+    MESSAGE = Column("events.message", "message", "transaction_name", "message", "message")
+    PLATFORM = Column("events.platform", "platform", "platform", "platform", "platform.name")
+    ENVIRONMENT = Column(
+        "events.environment", "environment", "environment", "environment", "environment"
+    )
+    RELEASE = Column(
+        "events.tags[sentry:release]", "tags[sentry:release]", "release", "release", "release"
+    )
+    DIST = Column("events.tags[sentry:dist]", "tags[sentry:dist]", "dist", "dist", "dist")
+    TITLE = Column("events.title", "title", "transaction_name", "title", "title")
+    TYPE = Column("events.type", "type", None, "type", "event.type")
+    TAGS_KEY = Column("events.tags.key", "tags.key", "tags.key", "tags.key", "tags.key")
+    TAGS_VALUE = Column("events.tags.value", "tags.value", "tags.value", "tags.value", "tags.value")
+    TAGS_KEYS = Column("events.tags_key", "tags_key", "tags_key", "tags_key", "tags_key")
+    TAGS_VALUES = Column("events.tags_value", "tags_value", "tags_value", None, "tags_value")
+    TRANSACTION = Column(
+        "events.transaction", "transaction", "transaction_name", "transaction", "transaction"
+    )
+    USER = Column("events.tags[sentry:user]", "tags[sentry:user]", "user", "user", "user")
+    USER_ID = Column("events.user_id", "user_id", "user_id", "user_id", "user.id")
+    USER_EMAIL = Column("events.email", "email", "user_email", "email", "user.email")
+    USER_USERNAME = Column("events.username", "username", "user_name", "username", "user.username")
+    USER_IP_ADDRESS = Column(
+        "events.ip_address", "ip_address", "ip_address_v4", "ip_address", "user.ip"
+    )
+    SDK_NAME = Column("events.sdk_name", "sdk_name", None, "sdk_name", "sdk.name")
+    SDK_VERSION = Column("events.sdk_version", "sdk_version", None, "sdk_version", "sdk.version")
+
     HTTP_METHOD = Column(
         "events.contexts[http.method]",
         "contexts[http.method]",
+        None,
         "contexts[http.method]",
         "http.method",
     )
     HTTP_REFERER = Column(
         "events.contexts[http.referer]",
         "contexts[http.referer]",
+        None,
```

Comment:
You could map to `contexts[http.referer]` here as transactions has a contexts column now. The other context values should work similarily.